management/src/main/java/org/ehcache/management/providers/ManagementProvider.java

```diff
@@ -81,9 +81,10 @@
    *
    * @param context the context.
    * @param statisticNames the statistic names to collect.
+   * @param since The unix time in ms from where to return the statistics for statistics based on samples.
    * @return the statistic values.
    */
-  <T extends Statistic<?>> Collection<T> collectStatistics(Map<String, String> context, String... statisticNames);
+  List<Statistic<?, ?>> collectStatistics(Map<String, String> context, Collection<String> statisticNames, long since);
```

Comment:
This one is a bit ugly. Since you're accepting a Collection of statisticNames, I fail to see why you're returning a List of statistic instead of a collection: the ordering cannot be guaranteed unless you pass a List.

It would have been preferable IMHO to move "long since" as the 2nd parameter and keep the String... statisticNames intact.